src/profile-logic/active-tab.js

```diff
@@ -127,9 +129,6 @@ export function computeActiveTabTracks(
   const mainTrackIndexesSet = new Set(mainTrackIndexes);
```

Comment:
Suggestion: This is a behavior change. Is it OK for this Set to be empty? To maintain old behavior, if it's empty, then the `0` index should be added. I'm not sure how this is used, but it would be good to ensure this doesn't regress or cause an error somewhere else.